pkg/querier/frontend/results_cache.go

```diff
@@ -107,6 +112,8 @@ func (s resultsCache) handleHit(ctx context.Context, r *QueryRangeRequest, exten
 		reqResps []requestResponse
 		err      error
 	)
+	log, ctx := spanlogger.New(ctx, "handleHit")
```

Comment:
Do we want to add the same line to `handleMiss`?

Reply:
I don't think so - the miss never modifies an existing extent.  In a hit we might, so we log the old trace ids.